pyaccel/elements.py

```diff
@@ -524,6 +525,18 @@ def kicktable(self, value):
             raise TypeError('value must be of Kicktable type')
         self.trackcpp_e.kicktable = value._kicktable
 
+    @property
+    def vchamber(self):
+        """Shape of vacuum chamber.
+        See trackcpp.VChamberShape for values."""
+        return self.trackcpp_e.vchamber
+
+    @vchamber.setter
+    def vchamber(self, value):
+        """Set shape of vacuum chamber.
+        See trackcpp.VChamberShape for values."""
+        self.trackcpp_e.vchamber = value
```

Comment:
maybe check if this value is non-negative?

Reply:
done